spec/units/recruitment_banner_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "govuk_web_banners/test_validators/recruitment_banner"
+require "govuk_web_banners/validators/recruitment_banner"
```

Comment:
Minor: it's odd that this needed to be updated rather than just added. I'm guessing something went awry with the rebase.

Reply:
It's needed, but it's in the wrong place. Have fixed

Reply:
(It's needed because we don't include the validators when we include the rest of the gem, because they're intended to only be present during dev/CI)